app/services/discussion_service.rb

```diff
@@ -28,6 +28,7 @@ def self.create(discussion:, actor:)
     return false unless discussion.valid?
 
     discussion.save!
+    DiscussionReader.for(discussion: discussion, user: actor).set_volume! :loud
```

Comment:
Is this a change in behaviour? Just trying to understand why we didn't need this before but we do now.